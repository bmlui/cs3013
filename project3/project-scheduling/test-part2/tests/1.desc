analysis: FIFO: with one job
