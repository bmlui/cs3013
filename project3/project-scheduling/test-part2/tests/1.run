./scheduler 1 FIFO tests/1.in 0
