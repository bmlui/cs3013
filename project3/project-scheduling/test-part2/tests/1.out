Execution trace with FIFO:
t=0: [Job 0] arrived at [0], ran for: [3]
End of execution with FIFO.
Begin analyzing FIFO:
Job 0 -- Response time: 0  Turnaround: 3  Wait: 0
Average -- Response: 0.00  Turnaround 3.00  Wait 0.00
End analyzing FIFO.
